project/.travis.yml.twig

```diff
@@ -53,7 +53,7 @@ matrix:
     - php: '{{ php|last }}'
       env: TARGET=lint
     - php: '{{ php|first }}'
-      env: COMPOSER_FLAGS="--prefer-lowest"
+      env: COMPOSER_FLAGS="--prefer-lowest" SYMFONY={{ versions.symfony|first }}.0
```

Comment:
as discussed, i agree with requiring the exact version for the lowest version. however, i am not sure if its a good idea to hardcode the `.0` - could we specify that in the config, or at least allow to override it? i have seen bundles that have e.g. symfony 2.8.18 as lowest version.

then again, if all sonata repos allow a .0 version in the lowest version, we can go with this and improve when we actually need it.

Reply:
I've tried to run build with `SYMFONY=2.8.*` and updated a table at the top of a page.

A result is the same as with `SYMFONY=2.8.0`! So i think this is the best option!

Reply:
Fixed